integration/kubernetes/helm-chart/alluxio/Chart.yaml

```diff
@@ -12,11 +12,13 @@
 name: alluxio
 apiVersion: v1
 description: Open source data orchestration for analytics and machine learning in any cloud.
-version: 0.6.25
+version: 0.6.26
 home: https://www.alluxio.io/
 maintainers:
 - name: Adit Madan
   email: adit@alluxio.com
+- name: Chris Zhu
+  email: czhu@alluxio.com
```

Comment:
@jiacheliu3 @madanadit @cheyang I hope you guys have no objections?

Reply:
nope :)